Penteract/Assets/Scripts/Fang.cpp

```diff
@@ -1,13 +1,14 @@
 #include "Fang.h"
 #include "GameplaySystems.h"
 #include "HUDController.h"
+#include "HUDManager.h"
 #include "CameraController.h"
 #include "UltimateFang.h"
 
-void Fang::Init(UID fangUID, UID trailGunUID, UID trailDashUID, UID leftGunUID, UID rightGunUID, UID rightBulletUID, UID leftBulletUID, UID cameraUID, UID canvasUID, UID EMPUID, UID EMPEffectsUID, UID fangUltimateUID, UID ultimateVFXUID) {
+void Fang::Init(UID fangUID, UID trailGunUID, UID trailDashUID, UID leftGunUID, UID rightGunUID, UID rightBulletUID, UID leftBulletUID, UID cameraUID, UID HUDManagerObjectUID, UID EMPUID, UID EMPEffectsUID, UID fangUltimateUID, UID ultimateVFXUID) {
 	SetTotalLifePoints(lifePoints);
 	characterGameObject = GameplaySystems::GetGameObject(fangUID);
-
+	ultimateTotalTime = 4.6f;
```

Comment:
Have this initialized either in the .h or in as assigning a constant, not a magic number

Reply:
The duration for both ultimates changes, Fang's is longer, in the .h we assign the duration of Onimaru's ultimate, a different approach could be taken, any suggestions?

Reply:
You can define a constexpr variable in Fang.h and assign it here to void magic numbers in the declaration of Fang.cpp